source/unit/Constraints.ooc

```diff
@@ -274,18 +287,18 @@ CompareWithinConstraint: class extends CompareConstraint {
 		this precision = precision as LDouble
 		this comparer = func (value, correct: Object) -> Bool { this testChild(value) }
 		f := func (value, correct: Object) -> Bool { (value as Cell<Float> get() - correct as Cell<Float> get()) abs() < precision }
-		CompareConstraint new(this, this correct, f, ComparisonType Within)
+		CompareConstraint new(this, Cell<Float> new(this correct as Cell<Float> get()), f, ComparisonType Within)
```

Comment:
`this correct` was shared between parent and several children, it was not clear who should delete it and was causing invalid delete / free after implementing `free` for `Modifier`.